docs/reference/method/MongoDBGridFSBucket-getChunksCollection.txt

```diff
@@ -28,4 +28,22 @@ Return Values
 
 A :phpclass:`MongoDB\\Collection` object for the chunks collection.
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   var_dump($bucket->getChunksCollection());
+
+The output would then resemble::
+
+   object(MongoDB\Collection)#3027 (2) {
+      ["collectionName"]=>
+      string(9) "fs.chunks"
+      ["databaseName"]=>
+      string(11) "phplib_test"
```

Comment:
This should be `string(4) "test"`.

This example output hides other fields in the debug output for the Collection, so I might suggest changing the example to dump `(string) $bucket->getChunksCollection()` instead and then the output can be:

```
string(14) "test.fs.chunks"
```